io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=~/Applications
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=io.github.prateekmedia.appimagepool
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    build-commands:
+      - cp -R appimagepool/usr/bin/ /app/appimagepool
+      - cp -R appimagepool/usr/share/ /app/share
+      - sed -i 's/<id>appimagepool/<id>io.github.prateekmedia.appimagepool/' /app/share/metainfo/appimagepool.appdata.xml
+      - mkdir -p /app/bin/ /app/lib/
+      - ln -s /usr/lib/${FLATPAK_ARCH}-linux-gnu/libgio-2.0.so.0 /app/lib/libgio-2.0.so
```

Comment:
Is it actually needed?

Reply:
Yes without it app fails to start, complaining libgio not found

Reply:
This was the build https://github.com/flathub/flathub/pull/2492#issuecomment-913108032 that fails to load libgio

Reply:
Ok removed it eventually.